main/src/ca/uwaterloo/flix/language/dbg/PrettyPrinter.scala

```diff
@@ -409,7 +409,7 @@ object PrettyPrinter {
             clazz.getName +
             methods.map(fmtJvmMethod(_, formatter)).mkString("{ ", " ", " }")
 
-        case Expression.Spawn(exp, tpe, loc) => "spawn " + visitExp(exp)
+        case Expression.Spawn(exp1, exp2, tpe, loc) => "spawn " + visitExp(exp1)
```

Comment:
Pretty print both exps.

Reply:
I left that out because I couldn't see anything else that printed its region argument. But I guess that's because it's discarded before this point in every other case 🤣

Do you have a feeling for the best format? Should I copy the `@r` syntax? Or...?

Reply:
Just copy the ` @ r` syntax for now.

Reply:
Done